test/controllers/timezone_controller_test.exs

```diff
@@ -9,9 +9,33 @@ defmodule Ask.TimezoneControllerTest do
     {:ok, conn: conn, user: user}
   end
 
-  test "returns Timex timezones", %{conn: conn} do
-    conn = get conn, timezone_path(conn, :timezones)
-    assert json_response(conn, 200)["timezones"] == Timex.timezones
-  end
+  describe "the timezones map" do
+    test "should link each deprecated timezone with its canonical timezone", %{conn: conn} do
```

Comment:
I would add a test for the specific case we're solving. Something like "should link `US/Eastern` with `America/New_York`"

Reply:
That case is covered by this test since is asserting all the non-canonical timezones